src/Arcus.Scripting.ARM/Scripts/Inject-ArmContent.ps1

```diff
@@ -3,15 +3,28 @@
     
     ${ fileToInject.xml }
     ${ FileToInject=file.xml }
+    ${ FileToInject=c:\file.xml }
     ${ FileToInject = ".\Parent Directory\file.xml" }
+    ${ FileToInject = "c:\Parent Directory\file.xml" }
     ${ FileToInject = ".\Parent Directory\file.xml", EscapeJson, ReplaceSpecialChars }
     ${ FileToInject = '.\Parent Directory\file.json', InjectAsJsonObject }
  #>
 
 param (
+
     [string] $Path = $PSScriptRoot
 )
 
+# [System.IO.Path]::IsPathFullyQualified() is not supported in PS 5.1 (as opposed to PS 7)
+function IsPathFullyQualified {
+    param(
+        [string] $pathToCheck
+    )
+
+    $rootInfo = [System.IO.Path]::GetPathRoot($pathToCheck)
+    return -not([string]::IsNullOrEmpty($rootInfo))

```

Comment:
Why do we make a distinction here between blank <> empty? Is this documented somehow that this indeed describes different results in IO?